docs/bundles/plain.md

```diff
@@ -29,16 +60,125 @@ manifests
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+└── deployment.yaml
+```
+
+If you are using [kustomize](https://kustomize.io/) for building your manifests from a template, redirect the output into a single file under the `manifests/` directory. For example:
+```bash
+$ tree templates
+templates
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+├── deployment.yaml.yaml
+└── kustomization.yaml
+```
+
+```bash
+kustomize build templates > manifests/manifests.yaml
+```
+
+### Git source
+
+For using the bundle with a [git source](../sources/git.md), commit your `manifests/` directory to your git repository.
+
+### HTTP source
+
+For using the bundle with a [HTTP source](../sources/http.md), create a `.tgz` file that contains the `manifests/` directory and it's manifests. This `.tgz` file should be served when hitting the HTTP endpoint.
+
+### Image source
+
+For using the bundle with an [image source](../sources/image.md), follow the below steps:
+
+1. Create a Dockerfile at the root of the project
+```bash
+touch Dockerfile.plainbundle
+```
+
+2. Edit the Dockerfile to include the following:
+```Dockerfile
+FROM scratch
+COPY manifests /manifests
+```
+
+> Note: The Dockerfile can have any `FROM ...` directive, but it is recommended to use the `FROM scratch`
+> directive to keep the resulting image size as minimal as possible
+
+3. Build the image using a container tool like docker or podman. Use an image tag that references a repository that you have push access to. For example,
+
+```bash
+docker build -f Dockerfile.plainbundle -t quay.io/operator-framework/rukpak:example .
+```
+
+4. Push the image to the remote registry
+
+```bash
+docker push quay.io/operator-framework/rukpak:example
+```
+
+If you are looking to use a private image registry for sourcing your bundle content, see the ["Private image registries" section of the image source documentation](../sources/image.md#private-image-registries)
+
+### Upload source
+
+For using the bundle with an [upload source](../sources/upload.md), follow the below steps:
+
+1. Download the `rukpakctl` CLI tool
+```bash
+go install github.com/operator-framework/rukpak/cmd/rukpakctl
+```
+
+2. Ensure rukpak is running. For more info on how to install rukpak on your cluster see the [Installation section of the README](../../README.md#installation)
+
+3. Use `rukpakctl` to create a `BundleDeployment` using the `manifests/` directory
+```bash
+rukpakctl run <bundle-name> .
+```
+
+### ConfigMap source
+
+For using the bundle with a [ConfigMap source (also known as a local source)](../sources/local.md), follow the below steps:
+
+1. Ensure rukpak is running. For more info on how to install rukpak on your cluster see the [Installation section of the README](../../README.md#installation)
+
+2. Create a `ConfigMap` from the `manifests/` directory
+```bash
+kubectl create configmap my-configmap --from-file=manifests -n rukpak-system
```

Comment:
```suggestion
$ kubectl create configmap my-configmap --from-file=manifests -n rukpak-system
```

Reply:
addressed in https://github.com/operator-framework/rukpak/pull/642/commits/486faf8915e2f3f891ee769b99651e3039d052b1